### PR TITLE
ELZT-72: initial commit of CapabilityStatement for the use case refer…

### PR DESCRIPTION
**Pull request by niekvangalen**
…ral from general practitioner to allied health professional

### CODE REVIEW

**Comment by niekvangalen** [comment]
@LuudSlagter this is a first approach for a minimum CapabilityStatement for both client and server, what do you think?

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <mode value="client"/>
+    <documentation value="Minimal requirements for a client to fulfill the 'Referral from general practitioner to allied health professional' usecase (system role: HZP-VPS)."/>
+     <resource>
+            <type value="Task"/>
+            <profile>
+                <reference value="http://nictiz.nl/fhir/StructureDefinition/hg-ReferralTask"/>
+                <display value="hg ReferralTask"/>
+            </profile>
+            <interaction>
+                <code value="search-type"/>
```
[the remark this answers is not in the document]
Comment (niekvangalen) [reply CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml:36]: No, not yet. Good point.

File: CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  <rest>
+    <mode value="client"/>
+    <documentation value="Minimal requirements for a client to fulfill the 'Referral from general practitioner to allied health professional' usecase (system role: HZP-VPS)."/>
+     <resource>
+            <type value="Task"/>
+            <profile>
+                <reference value="http://nictiz.nl/fhir/StructureDefinition/hg-ReferralTask"/>
+                <display value="hg ReferralTask"/>
+            </profile>
+            <interaction>
```
[the remark this answers is not in the document]
Comment (niekvangalen) [reply CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml:35]: Fixed.

File: CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  <kind value="requirements"/>
+  <fhirVersion value="4.0.1"/>
+  <format value="xml"/>
+  <format value="json"/>
+  <rest>
+    <mode value="client"/>
+    <documentation value="Minimal requirements for a client to fulfill the 'Referral from general practitioner to allied health professional' usecase (system role: HZP-VPS)."/>
+     <resource>
+            <type value="Task"/>
+            <profile>
```
[the remark this answers is not in the document]
Comment (niekvangalen) [reply CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml:31]: Fixed.

File: CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  <purpose value="This CapabilityStatement is informative in nature and does not represent the minimum or maximum set of capabilities the client or server should support. The aim is to design the CapabilityStatement as complete as possible, however for the exact set of capabilities the implementation guide of the corresponding information standard should be consulted."/>
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise."/>
+  <kind value="requirements"/>
+  <fhirVersion value="4.0.1"/>
+  <format value="xml"/>
+  <format value="json"/>
+  <rest>
+    <mode value="client"/>
+    <documentation value="Minimal requirements for a client to fulfill the 'Referral from general practitioner to allied health professional' usecase (system role: HZP-VPS)."/>
```
[the remark this answers is not in the document]
Comment (niekvangalen) [reply CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml:28]: Fixed.

File: CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  <rest>
+    <mode value="server"/>
+    <documentation value="Minimal requirements for a client to fulfill the 'Referral from general practitioner to allied health professional' usecase (system role: HZP-VPO)."/>
+              <resource>
+            <type value="Task"/>
+            <profile>
+                <reference value="http://nictiz.nl/fhir/StructureDefinition/hg-ReferralTask"/>
+                <display value="hg ReferralTask"/>
+            </profile>
+            <interaction>
```
[the remark this answers is not in the document]
Comment (niekvangalen) [reply CapabilityStatements/hg-Referral-GeneralPractitionerToAlliedHealthProfessional.xml:75]: I'm gonna leave search-type and read in here for now, to support future use cases.

**Comment by niekvangalen** [comment]
@LuudSlagter I processed your comments, can you take another look?

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**